Jenkinsfile

```diff
@@ -181,28 +184,84 @@ pipeline {
           }
         }
         stage('Release') {
-          options { skipDefaultCheckout() }
+          options {
+            skipDefaultCheckout()
+            timeout(time: 12, unit: 'HOURS')
```

Comment:
Wait for 12 hours when a release happens, that's only affecting the master branch